_posts/2023-12-13-release-0_4_0.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  "kroxylicious release 0.4.0"
+date:   2023-12-14 00:00:00 +1300
+author: "Keith Wall"
+author_url: "https://www.github.com/k-wall"
+categories:  releases kroxylcious topic-encryption
+---
+
+The Kroxylicious project is very pleased to announce the release of Kroxylicious 0.4.0.   The big news is this release includes our `EnvelopeEncryption` filter which provides an **Encryption-at-Rest** solution for the Apache Kafka(tm) which is _transparent_ to both clients and brokers.
```

Comment:
From a functional standpoint this is the big news, from a user standpoint I'd say the binary distribution is probably as big news.